{"name": "Тест загрузки МК", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u0426\\\\u041d\\\\u0422]_19_00_35.mp4'", "trace": "self = <allure_commons._allure.fixture object at 0x0000019C33319850>, args = ()\nkwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py:221: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\conftest.py:57: in record_video\n    allure.attach.file(outfile, attachment_type=allure.attachment_type.MP4)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\_allure.py:203: in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_pytest\\listener.py:233: in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\reporter.py:160: in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\n..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-0ztDBuEg-py3.11\\Lib\\site-packages\\allure_commons\\logger.py:53: in report_attached_file\n    shutil.copy2(source, destination)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py:436: in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nsrc = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u0426\\\\u041d\\\\u0422]_19_00_35.mp4'\ndst = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\tests\\\\allure-results\\\\11f7be1a-6cf9-4ae2-810f-8c99c184a3d2-attachment.mp4'\n\n    def copyfile(src, dst, *, follow_symlinks=True):\n        \"\"\"Copy data from src to dst in the most efficient way possible.\n    \n        If follow_symlinks is not set and src is a symbolic link, a new\n        symlink will be created instead of copying the file it points to.\n    \n        \"\"\"\n        sys.audit(\"shutil.copyfile\", src, dst)\n    \n        if _samefile(src, dst):\n            raise SameFileError(\"{!r} and {!r} are the same file\".format(src, dst))\n    \n        file_size = 0\n        for i, fn in enumerate([src, dst]):\n            try:\n                st = _stat(fn)\n            except OSError:\n                # File most likely does not exist\n                pass\n            else:\n                # XXX What about other special files? (sockets, devices...)\n                if stat.S_ISFIFO(st.st_mode):\n                    fn = fn.path if isinstance(fn, os.DirEntry) else fn\n                    raise SpecialFileError(\"`%s` is a named pipe\" % fn)\n                if _WINDOWS and i == 0:\n                    file_size = st.st_size\n    \n        if not follow_symlinks and _islink(src):\n            os.symlink(os.readlink(src), dst)\n        else:\n>           with open(src, 'rb') as fsrc:\nE           FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\IIFA_Economics_Auto-tests_v.0.3\\\\video_reports\\\\test_parser[\\\\u041c\\\\u041a_Upstream_v22.6\\\\u0411_6+6_\\\\u0426\\\\u041d\\\\u0422]_19_00_35.mp4'\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\shutil.py:256: FileNotFoundError"}, "description": "Тест запустил - Данил Подольный \nСтенд, на котором запускался тест - http://genrestest.nntc.pro/economics/", "steps": [{"name": "Прохождение авторизации на стенде", "status": "passed", "start": 1676289641742, "stop": 1676289641742}, {"name": "Переход в конструктор расчетов", "status": "passed", "start": 1676289641742, "stop": 1676289643063}, {"name": "Очистка мастер-книг со стенда", "status": "passed", "steps": [{"name": "Подчет всех МК на стенде", "status": "passed", "start": 1676289643081, "stop": 1676289648201}, {"name": "Удаление всех МК на стенде", "status": "passed", "start": 1676289648201, "stop": 1676289650633}, {"name": "Проверка остались ли МК на странице", "status": "passed", "start": 1676289650633, "stop": 1676289655720}, {"name": "F5 и повторная проверка", "status": "passed", "start": 1676289655720, "stop": 1676289662063}], "start": 1676289643063, "stop": 1676289662063}, {"name": "Загрузка мастер-книги на стенд", "status": "passed", "start": 1676289662063, "stop": 1676289977315}], "attachments": [{"name": "stdout", "source": "19abe036-c233-4390-b57f-91a809d967be-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "mb_name", "value": "'МК_Upstream_v22.6Б_6+6_ЦНТ'"}], "start": 1676289641742, "stop": 1676289977316, "uuid": "365bc9ad-0fa9-4ffb-925f-50d65f3c62ea", "historyId": "76c019581a308b900c8b57d0a9ccc719", "testCaseId": "649e79659ec5cf1411c309c586996b64", "fullName": "tests.test_calculation_constructor.TestCalculationConstructor#test_parser", "labels": [{"name": "severity", "value": "blocker"}, {"name": "feature", "value": "Загрузка МК на фронт"}, {"name": "story", "value": "Загрузка всех имеющихся МК на стенд и проверка, что все загрузилось корректно"}, {"name": "tag", "value": "calculacion_constructor"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_calculation_constructor"}, {"name": "subSuite", "value": "TestCalculationConstructor"}, {"name": "host", "value": "C100119"}, {"name": "thread", "value": "72316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_calculation_constructor"}]}